src/test/resources/com/puppycrawl/tools/checkstyle/filters/strategy/FileLength/Test/Test2.java

```diff
@@ -0,0 +1,14 @@
+package Checker.FileLength;
```

Comment:
Please add comments to all input files where you expect a violation. It's hard to always compare with expected.txt and patch.diff.
Something like: `// violation newline/patchedline` or `// violation newline` (depends on violation and strategy)

Reply:
done